azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,33 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot hub certificate root-authority"] = """
+        type: group
+        short-summary: Manage the certificate root-authority for an IoT Hub instance.
+    """
+
+    helps["iot hub certificate root-authority set"] = """
+        type: command
+        short-summary: Set the certificate root-authority for an IoT Hub instance to a specific version.
+        long-summary: Transition this resource to a certificate on the DigiCert Global G2 root or revert to Baltimore root.
```

Comment:
tiny nit, maybe we can add in parenthesis, the connection between a root certificate description and v2 or v1 i.e. `Transition this resource to a certificate on the DigiCert Global G2 root (v2) or revert to Baltimore root (v1).`